### DISCUSSION TITLE
Getting "Failed to run `config` for telescope.nvim" when trying to open NvChad Theme "selector"...

### DISCUSSION THREAD

**Discussion by Nartek01** [NvChad/NvChad#2398] [Q&A]
Trying to get NvChad working on Windows 11 Powershell but getting the error below after starting nvim -> selected "N" on default config and open theme selector "space + t +h".

The things I tried that resulted in failure:
Tried on latest Neovim
Tried on Neovim 0.9.0
Run NvChadUpdate and tried
Install fzf
Set mingw64 to path
Removed .../local/nvim after each try
Removed .../local/nvim-data after each try

Whole error:
```
Failed to run `config` for telescope.nvim
...a/lazy/telescope.nvim/lua/telescope/_extensions/init.lua:10: 'fzf' extension doesn't exist or isn't installed: ...nvim-data/lazy/telescope-fzf-native.nvim/lua/fzf_lib.lua:11: cannot load module 'C:/Users/narte/AppData/Local/nvim-data/lazy/telescope-fzf-native.nvim/lua/../build/libfzf.dll': The specified module could not be found.^M
# stacktrace:
  - telescope.nvim\lua\telescope\_extensions\init.lua:10 _in_ **load_extension**
  - telescope.nvim\lua\telescope\_extensions\init.lua:62 _in_ **load_extension**
  - AppData\Local\nvim/lua/plugins/init.lua:247 _in_ **config**
```

**Comment by siduck** [comment]
You dont need fzf. Just make should be there, through mingw. or try installing make from choco
